src/DDTrace/Tracer.php

```diff
@@ -172,18 +172,18 @@ public function startSpan($operationName, $options = [])
 
         $reference = $this->findParent($options->getReferences());
 
+        // avoid rounding errors, we only care about microsecond resolution here
+        $roundedStartTime = ($options->getStartTime() + 0.2) / 1000000;
```

Comment:
I realize you didn't change this, but why are we adding `.2` here? Do you know?

Reply:
because 48 bits of a double do not contain the 50 bits needed to make this precise